### PR TITLE
refactor: Add casta font files and text style

### PR DESCRIPTION
**Pull request by wal33d006**

### CODE REVIEW

File: packages/cosmos_ui_components/lib/cosmos_text_theme.dart [hunk 1]
```diff
@@ -4,6 +4,7 @@ abstract class CosmosTextTheme {
   factory CosmosTextTheme._() => throw UnsupportedError('Not meant to be instantiated');
 
   static const String fontInter = 'Inter';
+  static const String fontCasta = 'Casta';
```
Comment (wal33d006) [inline packages/cosmos_ui_components/lib/cosmos_text_theme.dart:7]: @andrzejchm 
Can't figure out how two separate font families should be handled, for now just added a new style with this new font family as a parameter to the property.

**Review by Zfinix — APPROVED** [review]

File: packages/cosmos_ui_components/lib/cosmos_text_theme.dart (hunk 1 above)
Comment (Zfinix) [reply packages/cosmos_ui_components/lib/cosmos_text_theme.dart:7]: Oh this is easy you just define a new family in the assets section of pubspec

Comment (wal33d006) [reply packages/cosmos_ui_components/lib/cosmos_text_theme.dart:7]: Yes I already did that, I am asking about injecting the family for different fonts dynamically

Comment (Zfinix) [reply packages/cosmos_ui_components/lib/cosmos_text_theme.dart:7]: 🤔 I don't understand

Comment (andrzejchm) [reply packages/cosmos_ui_components/lib/cosmos_text_theme.dart:7]: @wal33d006 this is a copyrighted font that should not be included in the public repo, its specific to the emeris app and should be placed there

Comment (Zfinix) [reply packages/cosmos_ui_components/lib/cosmos_text_theme.dart:7]: Oh yeah thats a draw back

**Pull request closed without merging**